<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>652</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblHeader">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>650</width>
        <height>120</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="core.qrc">:/resources/banner.jpg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblApplicationLine">
      <property name="text">
       <string>&lt;Application Line&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblTAppVersion">
        <property name="text">
         <string>Application Version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAppVersion">
        <property name="text">
         <string>&lt;App Version&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTQtVersion">
        <property name="text">
         <string>Qt Version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblQtVersion">
        <property name="text">
         <string>{Qt Version}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="core.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
